File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Admin_Update.ascx.cs
using NotificationSystem.NotificationSystem.Data.Classes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace NotificationSystem.NotificationSystem.Web.Controls_Search
{
	public partial class ctrSearch_Admin_Update : System.Web.UI.UserControl
	{
		protected void Page_Load(object sender, EventArgs e)
		{
			clsNotificationSystem theNotificationSystem = new clsNotificationSystem();

			if (!(Page.IsPostBack))
			{
				cmbAdmin.DataSource = theNotificationSystem.GetAdmins();
				cmbAdmin.DataTextField = "UserName"; cmbAdmin.DataValueField = "Password";
				cmbAdmin.DataBind();
			}
		}

		public void ClearControl()
		{
			this.cmbAdmin.Value = string.Empty; cmbAdmin.Text = "--";
		}
	}
}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Admin_Update.ascx.vb
Imports System
Imports System.Web.UI
Imports NotificationSystem.NotificationSystem.Data.Classes

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_Admin_Update
        Inherits UserControl
        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            Dim theNotificationSystem As New clsNotificationSystem

            If Not MyBase.Page.IsPostBack Then
                cmbAdmin.DataSource = theNotificationSystem.GetAdmins()
                cmbAdmin.DataTextField = "UserName"
                cmbAdmin.DataValueField = "Password"
                cmbAdmin.DataBind()
            End If
        End Sub

        Public Sub ClearControl()
            cmbAdmin.Value = String.Empty
            cmbAdmin.Text = "--"
        End Sub
    End Class
End Namespace
